src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -257,9 +259,18 @@ public void autonomousInit() {
     public void teleopInit() {
         if (SubsystemConstants.DRIVE_ENABLED) {
             subsystems.drivebaseSubsystem.setDefaultCommand(new DriveCommand(subsystems.drivebaseSubsystem,
-                    controls.driveController.getLeftXAxis(), controls.driveController.getLeftYAxis(),
+                    controls.driveController.getLeftYAxis(), controls.driveController.getLeftXAxis(),
                     controls.driveController.getRightXAxis()));
         }
+        if (!isCompetition())
+            return;
+        if (INTAKE_ENABLED) {
+            subsystems.intakeSubsystem.intakeExtend();
+        }
+        if (SHOOTER_ENABLED) {
```

Comment:
Check if shooterSubsystem is non-null vs. using something else to infer if it's valid or not.